.github/workflows/release.yaml

```diff
@@ -6,17 +6,24 @@ on:
   workflow_dispatch:
 
 jobs:
-  wait-on-test:
+  wait-on-tests:
     runs-on: ubuntu-latest
     steps:
-      - uses: jitterbit/await-check-suites@v1
+      - id: unit-tests
+        uses: fountainhead/action-wait-for-check@v1.0.0
         with:
-          timeoutSeconds: 600
-          # Only wait for the first check suite, so that only tests are awaited, and not eg crowdin.
-          onlyFirstCheckSuite: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: unit-tests
+      - id: cypress-tests
+        uses: fountainhead/action-wait-for-check@v1.0.0
+        with:
```

Comment:
do `token` and `uses: fountainhead/action-wait-for-check@v1.0.0` need to be repeated like this?

Reply:
the `checkName`s are different - this action is repeated for unit and cypress tests.